stellar/operators.js

```diff
@@ -152,19 +149,19 @@ if (require.main === module) {
     });
 
     program
-        .command('add_gas <sender> <messageId> <spender> <tokenAddress> <tokenAmount>')
-        .action((sender, messageId, spender, tokenAddress, tokenAmount, options) => {
-            mainProcessor(addGas, [sender, messageId, spender, tokenAddress, tokenAmount], options);
+        .command('collect_fees <receiver>')
```

Comment:
```suggestion
        .command('collect-fees <receiver>')
```
use kebab-case for CLI cmds everywhere

Reply:
ah, make sense. updated